test/models/student_test.rb

```diff
@@ -1,7 +1,26 @@
 require 'test_helper'
 
 class StudentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  def setup
+    @user = users(:one)
+    @s = students(:one)
+    # @s = Student.new(class_rank: "junior", room_draw_number: 53)
+  end
+
+
+  test "should be valid" do
+    assert @s.valid?, @s.errors.full_messages
```

Comment:
if you're going to add `@s.errors.full_messages` at this `valid?` check, you should add it everywhere. However, I don't think you need it. From what I can tell, it doesn't appear that people commonly add it to `assert` statements in Rails.